src/hooks/login/useLoginService.ts

```diff
@@ -123,7 +123,7 @@ export const useLoginService = (config?: OnProviderLoginType['nativeAuth']) => {
     }
     const messageToSign = new SignableMessage({
       address: new Address(address),
-      message: Buffer.from(`${address}${loginToken}{}`)
+      message: Buffer.from(`${address}${loginToken}`)
```

Comment:
Is it ok here? Is this the legacy support?

Reply:
yes, it was requested to be removed